bacon/django/static/bacon/css/bacon.css

```diff
@@ -33,6 +33,24 @@ div.bacon-toolbar {
 	margin-bottom: 0.5em;
 }
 
+div.bacon-dropper-activator {
+	display: inline-block;
```

Comment:
Only really thought of this after your comment, but you could use `display: flex` here

Reply:
Have tried this via the inspector just now but couldn't get it to work. Not sure if the containing box wants to be small and is squashing everything to re-flow, if it's because the other parts of the line aren't in a block container or what, but it doesn't seem like a trivial fix.

I guess `inline-block` is working because the surrounding elements are inline and the two don't want to mix.